src/mmw/apps/home/views.py

```diff
@@ -116,6 +116,10 @@ def set_url(layer):
 
 
 def get_client_settings(request):
+    # BiG-CZ mode applies when either ?bigcz=true query param present, or
+    # request host contains predefined host. This covers staging sites, etc.
+    bigcz = (request.GET.get('bigcz') == 'true' or
```

Comment:
I don't really know enough to be confident of how this plays out in the Python interpreter, but I think that it might be better to set the second condition first here. The way it stands now, in staging/production the `?bigcz=true` condition will always be evaluated first (and likely be false the vast majority of the time), which is slightly wasteful. Flipping the conditions should make staging/production have the shortest path to evaluation.